zerver/webhooks/stripe/view.py

```diff
@@ -149,8 +149,19 @@ def default_body(update_blacklist: List[str]=[]) -> str:
     if category == 'invoice':
         if event == 'upcoming':  # nocoverage
             body = 'Upcoming invoice created'
+        elif (event == 'updated' and
+              payload['data']['previous_attributes'].get('paid', None) is False and
+              object_['paid'] is True and
+              object_["amount_paid"] != 0 and
+              object_["amount_remaining"] == 0):
+            # We are taking advantage of logical AND short circuiting here since we need the else
+            # statement below.
+            object_id = object_['id']
+            invoice_link = 'https://dashboard.stripe.com/invoices/{}'.format(object_id)
+            body = '[Invoice]({invoice_link}) is now paid.'.format(invoice_link=invoice_link)
```

Comment:
let's go no period, since that's how we do the other strings in this integration

Reply:
Ok